addons/kotsadm/1.14.0-beta.2/install.sh

```diff
@@ -344,3 +347,31 @@ function kotsadm_namespaces() {
         kubectl create ns "$NAMESPACE" 2>/dev/null || true
     done
 }
+
+function kotsadm_health_check() {
+    if [[ -n $(kubectl get pods -l app=kotsadm -o jsonpath="{range .items[*]}{range .status.conditions[*]}{ .type }={ .status }{'\n'}{end}{end}" | grep Ready=False) ]]; then
```

Comment:
What if there are no pods?

Reply:
then we get the current 'no spinner' behavior, I suppose

Reply:
but pod creation should follow the deployment creation very quickly